lib/update_test.go

```diff
@@ -15,11 +15,14 @@ import (
 )
 
 func TestUpdateMethods(t *testing.T) {
+	prevDci := cron.DefaultCheckInterval
 	tmpDir, err := ioutil.TempDir("", "update_methods")
 	if err != nil {
 		t.Fatal(err)
```

Comment:
I could be wrong, but I thought t.Fatal exits. So the line after this wouldn't run, right?

Reply:
yep! Desired behaviour. Fatal exits the _test_, not the process. If we can't get a temp dir to write to, we shouldn't run this test